src/server/src/modules/student.ts

```diff
@@ -128,3 +127,52 @@ export const getJobApplications = async (studentId: number) => {
         ])
         .getMany();
 };
+
+export const searchStudents = async (
+    firstName: string,
+    lastName: string,
```

Comment:
replace types of the arguments of this function to use types from IStudent interface

Reply:
Done.